File: Daily/sbgo/Leetcode551-学生出勤记录.go
package sbgo

import "strings"

func checkRecord(s string) bool {
	if strings.Contains(s, "LLL") {
		return false
	}
	cnt := 0
	for _, ch := range s {
		if ch == 'A' {
			cnt++
		}
	}
	if cnt >= 2 {
		return false
	}
	return true
}

File: Leetcode/PY/Leetcode551-学生出勤记录.py
class Solution:
    def checkRecord(self, s: str) -> bool:
        if 'LLL' in s:
            return False
        cnt = 0
        for ch in s:
            if ch == 'A':
                cnt += 1
        if cnt >= 2:
            return False
        return True


# class Solution:
#     def checkRecord(self, s: str) -> bool:
#         return s.count('A') < 2 and not('LLL' in s)